src/discord/discord.service.ts

```diff
@@ -1,20 +1,20 @@
 import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
-import { CreateDiscordDto } from './dto/create-discord.dto';
 import { ReadDiscordDto } from './dto/read-discord.dto';
 import { UpdateDiscordDto } from './dto/update-discord.dto';
 
 @Injectable()
 export class DiscordService {
   private discord: ReadDiscordDto[] = [];
-  create(createDiscordDto: CreateDiscordDto) {
+  create(createDiscordDto: ReadDiscordDto) {
     const discordUser = {
       id: Date.now(),
       username: createDiscordDto.username,
-      bio: { ...createDiscordDto.bio },
+      bio: createDiscordDto.bio,
+      socials: { ...createDiscordDto.socials },
       createdOn: new Date(Date.now()),
       updatedOn: new Date(Date.now()),
     };
-    if (!discordUser.username) {
+    if (!discordUser.username || !discordUser.bio) {
```

Comment:
`bio` is not a required field, I think `username` is the only required field

Reply:
I made `bio` required field but I'll make it optional.